src/clappr-stats.js

```diff
@@ -234,10 +234,15 @@ export default class ClapprStats extends ContainerPlugin {
   }
 
   _calculatePercentages() {
-     if (this._metrics.extra.duration > 0) {
-       this._metrics.extra.bufferingPercentage = (this._metrics.timers.buffering / this._metrics.extra.duration) * 100
-     }
-  }
+    if(isNaN(this._metrics.extra.buffersize)){
+       this._metrics.extra.bufferingPercentage = NaN
+       return;
+    }
+
+    if (this._metrics.extra.duration > 0) {
+      this._metrics.extra.bufferingPercentage = (this._metrics.extra.buffersize / this._metrics.extra.duration) * 100
```

Comment:
@adiesel as @jhonatangomes mentioned, you're changing the calculation of the `bufferingPercentage`. This metric has a specific meaning, so if you need to add this new value, you should create another one.

Reply:
Makes sense. Will close the PR.